### PR TITLE
Settings: Add refresh rate preferences

### PR DESCRIPTION
**Pull request by clhexftw**
Settings: Add preference for KEY_MIN_REFRESH_RATE

Change-Id: Iac1f65ab09717ea55a5b471e094385c77ba894ee

Settings: Add peak refresh rate list preference

* AOSP 'Smooth display' setting is just a toggle, some devices support multiple refresh rates so add support for it with a ListPreference.

* @jhonboy121: unify config flag for both min and max list preference

Change-Id: I3da3d2b86e61ed3caf9af5770d8bdb4485817b97

Settings: Rewrite refresh rate preference controllers in kt and improve

* Added settings observer to update preference on settings changes. We have a tile so it would be nice to reflect changes when tile state is changed.
* Restrict max refresh rate list based on the min rate set since it is absurd to allow setting max values below current min rate.

Settings: Display 60 as min refresh rate if key value is 0

Settings: Fix refresh rate list preference values order

* basically just sort it before adding to list, apparently the order was not ascending for vayu

Settings: Update config for refresh rate

### CODE REVIEW

**Comment by franciscogire** [comment]
Olá  @clhexftw, fiz atualização da versão PixelOS,  android 14 no Poco X3 Pro,  mas não tem a opção de atualização, e não estou conseguindo inserir cartões na Carteira da Google,  Google Pay,  pode corrigir esse erro?
